hs_access_control/tests/test_create_resource_version.py

```diff
@@ -0,0 +1,118 @@
+                                                  )
+
+    def test_create_version(self):
+        # superuser/admin should not be able to create a resource version
+        self.assertFalse(self.admin.uaccess.can_create_resource_version(self.resource))
+
+        # ONLY owner should be able to create a resource version
+        self.assertTrue(self.owner.uaccess.can_create_resource_version(self.resource))
+
+        # grant editor edit permission
+        self.owner.uaccess.share_resource_with_user(self.resource, self.editor, PrivilegeCodes.CHANGE)
+        # resource editor can't create resource version
+        self.assertFalse(self.editor.uaccess.can_create_resource_version(self.resource))
+
+        # grant viewer view permission
+        self.owner.uaccess.share_resource_with_user(self.resource, self.viewer, PrivilegeCodes.VIEW)
+        # resource viewer can't create resource version
+        self.assertFalse(self.viewer.uaccess.can_create_resource_version(self.resource))
+
+        # authenticated user with no granted privilege can't create resource version
+        self.assertFalse(self.user.uaccess.can_create_resource_version(self.resource))
+
+    def test_create_version_inactive_user(self):
+        # inactive superuser/admin should not be able to create a resource version
+        self.admin.is_active = False
+        self.admin.save()
+        with self.assertRaises(PermissionDenied):
+            self.admin.uaccess.can_create_resource_version(self.resource)
+
+        # inactive resource owner should not be able to create a resource version
+        self.owner.is_active = False
+        self.owner.save()
+        with self.assertRaises(PermissionDenied):
+            self.owner.uaccess.can_create_resource_version(self.resource)
+
+        self.owner.is_active = True
+        self.owner.save()
+
+        # grant editor edit permission
+        self.owner.uaccess.share_resource_with_user(self.resource, self.editor, PrivilegeCodes.CHANGE)
+        # inactive editor should not be able to create a resource version
+        self.editor.is_active = False
+        self.editor.save()
+        with self.assertRaises(PermissionDenied):
+            self.editor.uaccess.can_create_resource_version(self.resource)
+
+        # grant viewer view permission
+        self.owner.uaccess.share_resource_with_user(self.resource, self.viewer, PrivilegeCodes.VIEW)
+        # inactive viewer should not be able to create a resource version
+        self.viewer.is_active = False
+        self.viewer.save()
+        with self.assertRaises(PermissionDenied):
+            self.viewer.uaccess.can_create_resource_version(self.resource)
+
+        # inactive user should not be able to create a resource version
+        self.user.is_active = False
+        self.user.save()
+        with self.assertRaises(PermissionDenied):
+            self.user.uaccess.can_create_resource_version(self.resource)
```

Comment:
@hyi If you look at unit tests for access control you will see inactive user is tested for all access control public functions (Alva's code). I have done the same thing for this new function that I added. If testing is through I think it is good. In order to make sure that only owners can create version we have to test that no one else can create version.